usr/share/rear/build/default/985_fix_broken_links.sh

```diff
@@ -0,0 +1,11 @@
+# build/default/985_fix_broken_links.sh
+# Check for broken symbolic links in our ROOTFS_DIR area - issue #1638
+
+pushd $ROOTFS_DIR >/dev/null
+    find . -xtype l | grep -v "/dev/" | while read SYMLINK
```

Comment:
How will that find symlinks that point outside ROOTFS_DIR (esp. absolute symlinks)? They are not broken now, but will be when the rescue system boots.

Reply:
To detect this the find command should actually run in a chroot IMO.

Reply:
@pcahyna 
can you explain why find may not work here
when it runs inside the ROOTFS_DIR directory?